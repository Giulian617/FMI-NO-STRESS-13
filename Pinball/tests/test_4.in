100 0
22 16 7
8 22 8
64 16 10
95 64 8
56 95 3
4 64 7
79 8 6
91 95 5
55 64 3
72 16 9
59 16 2
96 64 5
51 55 9
100 79 9
49 64 3
30 91 2
68 30 10
70 91 1
34 91 8
23 34 9
44 64 4
98 70 4
53 96 1
67 53 2
28 23 7
90 44 7
71 30 7
24 67 5
66 79 7
94 30 1
43 56 5
39 90 6
77 68 1
61 49 4
57 100 7
82 39 5
69 91 5
60 55 1
52 56 8
13 8 9
85 57 3
78 39 7
26 72 3
1 22 2
76 30 5
46 77 7
87 23 7
2 76 8
75 49 10
80 67 4
41 57 5
33 55 3
9 34 8
89 76 6
99 75 9
92 66 5
11 34 1
20 100 4
32 85 5
63 4 7
31 2 6
35 55 3
83 89 5
74 2 7
65 100 9
45 20 2
36 24 10
54 72 1
19 52 8
38 22 10
97 22 2
81 67 1
15 13 5
93 72 5
84 74 7
40 83 4
42 94 9
88 79 10
86 98 6
21 95 6
48 59 8
18 33 2
12 44 8
25 16 5
17 88 6
73 31 10
47 73 2
27 63 1
50 12 8
3 67 7
14 49 3
5 65 10
37 46 10
29 85 5
58 76 9
10 79 7
7 27 10
62 72 9
6 7 1
