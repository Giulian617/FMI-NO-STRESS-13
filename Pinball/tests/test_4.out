0
776576733
10909091
40360874
824522534
142312010
853065322
313976875
577777782
631874303
521100431
662283590
652421657
229344731
168803420
838793928
605757580
605757580
521100431
983269098
321214751
703703709
737549755
352525255
732121480
163927242
893737380
69261385
891737898
450395698
563301286
722934478
606324629
840366815
751969169
506410260
96984806
124850325
584056140
586615277
396653821
560040101
247471617
419396964
838793928
142312010
614672369
740740746
379922912
